src/credentials/attestation_verifier/DefaultDeviceAttestationVerifier.cpp

```diff
@@ -607,12 +613,147 @@ CHIP_ERROR DefaultDACVerifier::VerifyNodeOperationalCSRInformation(const ByteSpa
     return CHIP_NO_ERROR;
 }
 
+// This method parses the below JSON Scheme
```

Comment:
So right now, you can provide a trust store to a DefaultDACVerifier and it will work with that via API, agnostic to how the data is actually stored or gotten.

We should probably do something similar here; there are existing deployments of DefaultDACVerifier that cannot in fact create a file on disk to point it to, so that detail should be hidden behind an API boundary, with some TestSomething class that implements the relevant API and has the JSON thing as an implementation.

The API to that thing should ideally be async, just like the API to the DefaultDACVerifier itself.